src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,143 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double MULTIPLIER = 1.5;
+    private int size;
+    private Node[] elements;
```

Comment:
That's NOT how you implement a linked list, it's conceptually different approach than arraylist, which is based on internal array and a managing wrapper. Please revisit theory on this subject, and don't hesitate to ask for guidance in Mate chat